reference/src/glossary.md

```diff
@@ -2,14 +2,15 @@
 
 #### Interior mutability
 
-*Interior Mutability* means mutating memory where there also exists a live shared reference immediately (i.e., non-transitively) pointing to the same memory.
-This propagates recursively through references, but not through raw pointers.
+*Interior Mutability* means mutating memory where there also exists a live shared reference pointing to the same memory; or mutating memory through a pointer derived from a shared reference.
```

Comment:
grammatical nit: I believe this is a definition of "interior **mutation**". I guess interior **mutability** is the ability to do interior mutation without it being UB.

Reply:
Better like this?